### ISSUE TITLE
Twig: Accordion items steal focus even when they're not visible

### ISSUE THREAD

**Issue by justintemps** [international-labour-organization/designsystem#846]
## The problem

You can reproduce this using Tab navigation on this page: https://www-preview.ilo.org/about/newsroom/test-multimedia-story

When the user is tabbing, eventually they get to the accordion items. But after the first accordion item, focus disappears inside its children even if they're not visible. This means that to tab through all of the tabs, a user would have to tab through all of the options in all of the Accordion tabs even if they aren't visible.

Here's a screen recording to illustrate the problem

https://github.com/international-labour-organization/designsystem/assets/12401179/858f587d-29da-46f3-8c70-3b7b646a1ec1

## The Solution

Instead, when an Accordion item is closed, it's children should not be able to get focus. A user tabbing through the Accordion should go Accordion Tab to Accordion Tab. Only when the Accordion is open should its children be able to get focus.

**Comment by justintemps** [comment]
This was fixed in #867
